docs/data/sql_functions.yml

```diff
@@ -679,9 +679,15 @@ temporal:
   - sql: TO_DATE(string1[, string2])
     table: toDate(STRING1[, STRING2])
     description: Converts a date string string1 with format string2 (by default 'yyyy-MM-dd') to a date.
-  - sql: TO_TIMESTAMP_LTZ(numeric, precision)
+  - sql: TO_TIMESTAMP_LTZ(numeric[, precision])
```

Comment:
Each time there is any update in doc, Chinese version also should be updated in same way.
It could be updated with same content in English however have to be kept in sync

Reply:
added doc to `sql_functions_zh.yml`